pywt/_multidim.py

```diff
@@ -31,6 +31,10 @@ def dwt2(data, wavelet, mode='symmetric'):
         Wavelet to use
     mode : str, optional
         Signal extension mode, see Modes (default: 'symmetric')
+    axes : 2-tuple of ints, optional
+        Axes over which to compute the DWT. Repeated elements mean the DWT will
+        be performed multiple times along these axes. A value of `None` (the
```

Comment:
Documentation is at odds with implementation. State the restriction to two axes if that is being used.